docs/ide/tutorial-csharp-console.md

```diff
@@ -18,7 +18,7 @@ ms.workload:
 ---
 # Tutorial: Get started with a C# console app in Visual Studio
 
-In this tutorial for C#, you'll use Visual Studio to create and run a console app and explore some features of the [Visual Studio integrated development environment (IDE)](visual-studio-ide.md) while you do so.
+In this tutorial for C#, you'll use Visual Studio to create and run a console app while exploring some features of the [Visual Studio integrated development environment (IDE)](visual-studio-ide.md).
```

Comment:
@kallerdaller We purposefully avoid words ending in "-ing" because they translate oddly into some languages. Respectfully, I decline this edit.